pkgs/top-level/python-packages.nix

```diff
@@ -4344,6 +4344,10 @@ in {
 
   mlxtend = callPackage ../development/python-modules/mlxtend { };
 
+  mlt-bindings = callPackage ../development/python-modules/mlt-bindings {
+    mlt = (callPackage ../development/libraries/mlt { python = self.python; enablePython = true; });
```

Comment:
Should we move this to top-level.nix?

Reply:
I've refactored the Python3 MLT bindings thanks to the answer of jtojnar here:

https://discourse.nixos.org/t/libxml-not-found-with-dependent-build/13603/5

The `development/python-modules/mlt-bindings` package definiton was removed altogether.